code/modules/surgery/dental_implant.dm

```diff
@@ -0,0 +1,52 @@
+/datum/surgery/dental_implant
+	name = "dental implant"
+	steps = list(/datum/surgery_step/generic/drill, /datum/surgery_step/insert_pill)
+	possible_locs = list("mouth")
+
+/datum/surgery/dental_implant/can_start(mob/user, mob/living/carbon/target)
+	if(istype(target,/mob/living/carbon/human))
+		var/mob/living/carbon/human/H = target
+		if(!H.check_has_mouth())
+			return 0
+		return 1
+
+/datum/surgery_step/insert_pill
+	name = "insert pill"
+	allowed_tools = list(/obj/item/weapon/reagent_containers/food/pill = 100)
```

Comment:
Yeah, you could use a patch, instead--dunno why you'd want to, though, as it hold 10 units less.